plugin/evm/block_verification.go

```diff
@@ -109,23 +108,6 @@ func (v blockValidator) SyntacticVerify(b *Block, rules params.Rules) error {
 		return fmt.Errorf("invalid mix digest: %v", ethHeader.MixDigest)
 	}
 
-	// Enforce static gas limit after ApricotPhase1 (prior to ApricotPhase1 it's handled in processing).
-	if rules.IsCortina {
-		if ethHeader.GasLimit != cortina.GasLimit {
-			return fmt.Errorf(
-				"expected gas limit to be %d after cortina but got %d",
-				cortina.GasLimit, ethHeader.GasLimit,
-			)
-		}
-	} else if rules.IsApricotPhase1 {
-		if ethHeader.GasLimit != ap1.GasLimit {
-			return fmt.Errorf(
-				"expected gas limit to be %d after apricot phase 1 but got %d",
-				ap1.GasLimit, ethHeader.GasLimit,
-			)
-		}
-	}
-
```

Comment:
With ACP-176, validating the gasLimit will require the parent header. This verification is already being performed in `consensus.go`, so removing this seemed to be the best approach.